wpk/windows/Dockerfile

```diff
@@ -4,10 +4,12 @@ RUN apt-get update && \
     apt-get -y install \
       python \
       git \
-      curl && \
+      curl \
+      jq && \
     curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
     python get-pip.py && \
-    pip install --upgrade cryptography 
+    pip install --upgrade cryptography && \
+    pip install awscli
```

Comment:
```suggestion
    pip install --upgrade cryptography awscli && \
```

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/467/commits/1c82879d25119ef9aeb3808e1a4f12747d8edbf0